docs/content/guides/recipes/howto-private-registry/snippets/env-complete.bicep

```diff
@@ -0,0 +1,54 @@
+//SECRETSTORE
+import radius as radius
+
+@description('Required value, refers to the personal access token or password of the git platform')
+@secure()
+param pat string
+
+resource secretStoreGit 'Applications.Core/secretStores@2023-10-01-preview' = {
+  name: 'my-git-secret-store'
+  properties: {
+    type: 'generic'
+    data: {
+      pat: {
+        value: pat 
+      }
+    }
+  }
+}
+//SECRETSTORE
+
+//ENV
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+  name: 'my-env'
+  properties: {
+    compute: {
+      kind: 'kubernetes'
+      namespace: 'my-namespace'
+    }
+    recipeConfig: {
+      terraform: {
+        authentication: {
+          git: {
+            pat: {
+              // The hostname of your git platform, such as 'dev.azure.com' or 'github.com'
+              'github.com':{
+                secret: secretStoreGit.id
+              }
+            }
+          }
+        }
+      }
+    }
+    recipes: {
+      'Applications.Datastores/redisCaches': {
+        default: {
+          templateKind: 'terraform'
+          // Git template path
+          templatePath:'git::https//github.com/my-org/my-repo'
```

Comment:
This doesn't look quite right. Could you check that the format is correct for the URL?